AstarteDeviceSDKCSharp/AstartePairingService.cs

```diff
@@ -39,11 +39,18 @@ public class AstartePairingService
         private readonly string _astarteRealm;
         private readonly HttpClient _httpClient;
 
-        public AstartePairingService(string pairingUrl, string astarteRealm, TimeSpan timeOut)
+        public AstartePairingService(string pairingUrl, string astarteRealm, TimeSpan timeOut, bool ignoreSSLErrors = false)
         {
             _astarteRealm = astarteRealm;
             _pairingUrl = new Uri($"{pairingUrl.TrimEnd('/')}/v1");
-            _httpClient = new HttpClient
+            var httpHandler = new HttpClientHandler();
+
+            if (ignoreSSLErrors)
+            {
+                httpHandler.ServerCertificateCustomValidationCallback = (sender, cert, chain, sslPolicyErrors) => true;
+            }
+
+            _httpClient = new HttpClient(httpHandler)
```

Comment:
Since httpHandler is used only if ignoreSSLErrors = true, What do you think to move the definition into the if block?
```suggestion

            if (ignoreSSLErrors)
            {
                var httpHandler = new HttpClientHandler();
                httpHandler.ServerCertificateCustomValidationCallback = (sender, cert, chain, sslPolicyErrors) => true;
                _httpClient = new HttpClient(httpHandler)
            }else {
                _httpClient = new HttpClient
            }

            ```

Reply:
Changed.